tasks/main.yml

```diff
@@ -40,3 +40,15 @@
 - name: Start kubelet
   become: yes
   service: name=kubelet.service state=started enabled=yes
+
+- name: make fstab directory for cifs plugin
+  become: yes
+  file:
+    path: "{{kubelet_volume_plugin_dir}}/fstab~cifs"
+    state: directory
+  when: kubelet_enable_cifs_mount
+
+- name: install cifs plugin
```

Comment:
Does Kubelet not need to be restarted for this? If it does, then these tasks should call the appropriate handler.

Reply:
Puneet, we don't need to restart Kubelet for this. The presence of this file on the nodes itself, will allow pods to mount CIFS volume.